server/schedule/checker/replica_checker.go

```diff
@@ -30,11 +30,27 @@ import (
 
 const (
 	replicaCheckerName = "replica-checker"
+	replicaChecker     = "replica_checker"
```

Comment:
no used?

Reply:
in line 38~53

Reply:
replicaCheckerName and replicaChecker, can we just keep one? I think the metrics are not a big compatibility issue.

Reply:
There are some similar problem in other scheduler and checker.I think we can uniform them in other pr.